parsec/core/gui/file_items.py

```diff
@@ -26,7 +27,10 @@ class FileType(IntEnum):
 
 
 class CustomTableItem(QTableWidgetItem):
-    def __lt__(self, other):
+    def __lt__(self, other: object) -> bool:
```

Comment:
Why not type `other` as `CustomTableItem` directly?

Reply:
I typed this parameter as `object` because mypy complains that `QTableWidgetItem` has `__lt__` defined like this `def __lt__(self, other: object) -> bool: ...`. 

If `other` is typed `CustomTableItem` here we'll get the following error:
```
parsec/core/gui/file_items.py:30: error: Argument 1 of "__lt__" is incompatible with supertype "QTableWidgetItem"; supertype defines the argument type as "object"  [override]
parsec/core/gui/file_items.py:30: note: This violates the Liskov substitution principle
parsec/core/gui/file_items.py:30: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides
```

Reply:
👌